README.md

```diff
@@ -1 +1,23 @@
-# Hello_micorverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+## Hello_Micorverse 
+
+This is the first project of Microverse
+
+## Built With
+
+- Html
+- Css
+
+
+
+
```

Comment:
- [ ] Great job documenting your code! 👍🏼 However, as stated by the last code reviewer, in order to provide professional and descriptive documentation, you could add some more information to your `README` file 🗒️  such as a `Getting started` section 🔧 , describing the steps that other people need to follow in order to get you code set and running. A `Contributing` section, describing how others can contribute to your project (usually, using issues). A `Support` section motivating people to support your project, and so on. 💯 

You're almost done! Please work on these minor changes and also take them in mind for your future projects! 🛠️